Syntaxes/Smarty.tmLanguage

```diff
@@ -128,9 +128,14 @@
 							<key>name</key>
 							<string>variable.parameter.smarty</string>
 						</dict>
+						<key>2</key>
+						<dict>
+							<key>name</key>
+							<string>keyword.operator.assignment.smarty</string>
+						</dict>
 					</dict>
 					<key>match</key>
-					<string>\b([a-zA-Z]+)=</string>
+					<string>\b([a-zA-Z]+)(=)</string>
```

Comment:
Though Sublime doesn't care about it, would you mind updating the regex in `Syntaxes/Smarty.JSON-tmLanguage` to mirror the addition of the capture group?